include/Util/BasicTypes.h

```diff
@@ -209,7 +206,42 @@ class SVFFunction : public SVFValue
         this->reachableBBs = reachableBBs;
     }
 
-    inline const bool isNotRetFunction() const
+    inline const Map<const BasicBlock*,Set<const BasicBlock*>>& getDomFrontierMap() const
+    {
+        return dfBBsMap;
+    }
+
+    inline Map<const BasicBlock*,Set<const BasicBlock*>>& getDomFrontierMap()
+    {
+        return dfBBsMap;
+    }
+
+    inline const Map<const BasicBlock*,Set<const BasicBlock*>>& getPostDomTreeMap() const
+    {
+        return pdtBBsMap;
+    }
+
+    inline Map<const BasicBlock*,Set<const BasicBlock*>>& getPostDomTreeMap()
+    {
+        return pdtBBsMap;
+    }
+    
+    inline Map<const BasicBlock*,Set<const BasicBlock*>>& getDomTreeMap()
+    {
+        return dtBBsMap;
+    }
+
+    inline const Map<const BasicBlock*,Set<const BasicBlock*>>& getDomTreeMap() const
+    {
+        return dtBBsMap;
+    }
+
+    inline bool isUnreachable(const BasicBlock* bb) const 
+    {
+        return std::find(reachableBBs.begin(), reachableBBs.end(), bb)==reachableBBs.end();
```

Comment:
Can use `reachableBBs.find` -- easier to read.

Reply:
reachableBBs that returned from LLVMUtil is a vector rather than a set. Thus, it does not have the find function.

Reply:
Ah got it, makes sense